File: lib/src/main/java/lib/Action.java
package lib;

public enum Action {

    /**
     * Register user on server. Use to save user info on server
     */
    USER_REGISTER,
    /**
     * Send to other party public key of session
     */
    SESSION_UPDATE,
    /**
     * Request session updates from server
     */
    SESSION_GET,
    /**
     * Send encrypted message
     */
    ENVELOPE,
    /**
     * Receive encrypted messages from server
     */
    ENVELOPE_GET

}

File: lib/src/main/kotlin/lib/Action.kt
package lib

enum class Action {
    /**
     * Register user on server. Use to save user info on server
     */
    USER_REGISTER,

    /**
     * Send to another party public key of session
     */
    SESSION_UPDATE,

    /**
     * Request session updates from server
     */
    SESSION_GET,

    /**
     * Send an encrypted message
     */
    ENVELOPE,

    /**
     * Receive encrypted messages from server
     */
    ENVELOPE_GET
}
